### PR TITLE
Dashboard learn section fixes

### PR DESCRIPTION
**Pull request by herflis**
Additionally I fixed all occurences of sensenet with 'S'

### CODE REVIEW

**Review by pusztaienike — APPROVED** [review]
👽

**Review by taki9 — CHANGES_REQUESTED** [review]
> buttons should have the same labels as seen on design

**Review by huszerlbence — APPROVED** [review]
checked on the test site and everything seems good to me.
thanks

File: apps/sensenet/src/components/field-controls/webhook-payload.tsx [hunk 1]
```diff
@@ -106,35 +120,15 @@ export const WebhookPayload: React.FC<ReactClientFieldSetting<LongTextFieldSetti
...
-            cols={70}
-            value={value}
-          />
+          </Typography>
+          <Typography variant="body1" gutterBottom={true}>
+            {useDefault ? localization.webhooksPayload.defaultPayload : localization.webhooksPayload.customize}
+          </Typography>
+          <Paper elevation={0} className={widgetClasses.container}>
+            {<pre className={classes.defaultPayload}>{value}</pre> || ''}
+          </Paper>
```
Comment (taki9) [inline apps/sensenet/src/components/field-controls/webhook-payload.tsx:131]:
```tsx
{useDefault ? 
  <Typography variant="body1" gutterBottom={true}>
    {localization.webhooksPayload.defaultPayload} 
  </Typography>
:
  <Paper elevation={0} className={widgetClasses.container}>
    <pre className={classes.defaultPayload}>{value}</pre>
  </Paper>
}
```

- rename classes.defaultPayload to classes.payload (it is the customized one)
- remove localization.webhooksPayload.customize

**Review by taki9 — CHANGES_REQUESTED** [review]

**Review by taki9 — APPROVED** [review]

**Pull request merged**